samples/caf_sensor_manager/remote/sensor_stub_overlay.conf

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+################################################################################
+# Application configuration overlay to select Sensor Stub
+
+# Select Sensor Stub
+CONFIG_SENSOR_SIM=n
+CONFIG_SENSOR_STUB=y
```

Comment:
Do we need this file both for app and remote core? Cannot we have it once and used twice?

Reply:
We could, but then the build command would be more complex. Again - no strong opinion here.

Reply:
Again It's just a suggestion if you think it is ok leave it as it is. :)